Error on import

In a new npm project :
```ts
import { OdooClient } from 'odoo-xmlrpc-ts';

const client = new OdooClient({
    url: 'http://localhost:8069',
    db: 'odoo',
    username: 'admin',
    password: 'admin',
});
```
Results :
```
node:internal/modules/esm/resolve:275
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/tmp/sdkjfsjdfj/node_modules/odoo-xmlrpc-ts/dist/esm/client' imported from /tmp/sdkjfsjdfj/node_modules/odoo-xmlrpc-ts/dist/esm/index.js
    at finalizeResolution (node:internal/modules/esm/resolve:275:11)
    at moduleResolve (node:internal/modules/esm/resolve:860:10)
    at defaultResolve (node:internal/modules/esm/resolve:984:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:685:12)
    at #cachedDefaultResolve (node:internal/modules/esm/loader:634:25)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:617:38)
    at ModuleLoader.getModuleJobForImport (node:internal/modules/esm/loader:273:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:135:49) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///tmp/sdkjfsjdfj/node_modules/odoo-xmlrpc-ts/dist/esm/client'
}

Node.js v22.14.0
```

Same error here on node 22

```
node:internal/modules/esm/resolve:257
    throw new ERR_MODULE_NOT_FOUND(
          ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module 'C:\redacted\node_modules\odoo-xmlrpc-ts\dist\esm\client' imported from C:\redacted\node_modules\odoo-xmlrpc-ts\dist\esm\index.js
    at finalizeResolution (node:internal/modules/esm/resolve:257:11)
    at moduleResolve (node:internal/modules/esm/resolve:914:10)
    at defaultResolve (node:internal/modules/esm/resolve:1039:11)
    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:554:12)
    at ModuleLoader.resolve (node:internal/modules/esm/loader:523:25)
    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:246:38)
    at ModuleJob._link (node:internal/modules/esm/module_job:126:49) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///C:/redacted/node_modules/odoo-xmlrpc-ts/dist/esm/client'
}

Node.js v22.6.0
```

I'll try to open a PR to fix it later today.

@iraycd sorry for the late reply, missed the notification. Here's the full error:

```
ℹ Error: Cannot find module '.\node_modules\odoo-xmlrpc-ts\dist\esm\client' imported from .\node_modules\odoo-xmlrpc-ts\dist\esm\index.js

 ⁃ at finalizeResolution (node:internal/modules/esm/resolve:264:11)
 ⁃ at moduleResolve (node:internal/modules/esm/resolve:924:10)
 ⁃ at defaultResolve (node:internal/modules/esm/resolve:1148:11)
 ⁃ at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:528:12)
 ⁃ at ModuleLoader.resolve (node:internal/modules/esm/loader:497:25)
 ⁃ at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:231:38)
 ⁃ at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:97:39)
 ⁃ at link (node:internal/modules/esm/module_job:96:36)

[CAUSE]
Error {
  stack: "Cannot find module 'A:\\redacted\\node_modules\\odoo-xmlrpc-ts\\dist\\esm\\client' imported from A:\\redacted\\node_modules\\odoo-xmlrpc-ts\\dist\\esm\\index.js\n" +
  '    at finalizeResolution (node:internal/modules/esm/resolve:264:11)\n' +
  '    at moduleResolve (node:internal/modules/esm/resolve:924:10)\n' +
  '    at defaultResolve (node:internal/modules/esm/resolve:1148:11)\n' +
  '    at ModuleLoader.defaultResolve (node:internal/modules/esm/loader:528:12)\n' +
  '    at ModuleLoader.resolve (node:internal/modules/esm/loader:497:25)\n' +
  '    at ModuleLoader.getModuleJob (node:internal/modules/esm/loader:231:38)\n' +
  '    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:97:39)\n' +
  '    at link (node:internal/modules/esm/module_job:96:36)',
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///A:/redacted/node_modules/odoo-xmlrpc-ts/dist/esm/client',
  message: "Cannot find module
  'A:\\redacted\\node_modules\\odoo-xmlrpc-ts\\dist\\esm\\client'
  imported from
  A:\\redacted\\node_modules\\odoo-xmlrpc-ts\\dist\\esm\\index.js",
}
```

--- a/jest.config.cjs
+++ b/jest.config.cjs
@@ -8,4 +8,8 @@ module.exports = {
   coverageDirectory: 'coverage',
   coveragePathIgnorePatterns: ['/node_modules/', '/test/'],
   testTimeout: 10000,
+  moduleNameMapper: {
+    '^(\\.{1,2}/.*)\\.js$': '$1',
+  },
+  extensionsToTreatAsEsm: ['.ts'],
 };

--- a/src/client.ts
+++ b/src/client.ts
@@ -1,5 +1,5 @@
 // src/client.ts
-import * as xmlrpc from 'xmlrpc';
+import xmlrpc from 'xmlrpc';
 
 import {
   OdooConfig,
@@ -8,8 +8,8 @@ import {
   SearchOptions,
   SearchReadOptions,
   OdooDomain,
-} from './types';
-import { OdooError, OdooAuthenticationError } from './errors';
+} from './types.js';
+import { OdooError, OdooAuthenticationError } from './errors.js';
 
 export class OdooClient {
   private common: xmlrpc.Client;

--- a/src/index.ts
+++ b/src/index.ts
@@ -1,3 +1,3 @@
-export * from './client';
-export * from './types';
-export * from './errors';
+export * from './client.js';
+export * from './types.js';
+export * from './errors.js';

--- a/tsconfig.esm.json
+++ b/tsconfig.esm.json
@@ -1,7 +1,8 @@
 {
   "extends": "./tsconfig.base.json",
   "compilerOptions": {
-    "module": "es2020",
-    "outDir": "./dist/esm"
+    "module": "NodeNext",
+    "outDir": "./dist/esm",
+    "moduleResolution": "nodenext"
   }
 }

